src/agent/dtls_mbedtls.cpp

```diff
@@ -77,8 +77,31 @@ static void MbedtlsDebug(void *aContext, int aLevel, const char *aFile, int aLin
         aLevel = OTBR_LOG_DEBUG;
         break;
     }
+    char buf[100];
+    /* mbed inserts a EOL
+     * And so does the otbrLog()
```

Comment:
In the end, there are blank lines between logs - that is why they are removed.
I believe syslog - quietly removes them, they are not removed when this goes to the seperate file